src/utils/urls.js

```diff
@@ -0,0 +1,14 @@
+export const getLoginUrl = () => (
+  `${process.env.REACT_APP_EGGHEAD_BASE_URL}/users/jwt?return_to=${window.location.href}`
+)
+
+export const guideUrl = 'https://instructor.egghead.io/guide'
+
+export const chatUrl = 'https://eggheadio.slack.com/messages/egghead-instructors/'
```

Comment:
We've got a Slack bot that can do amazing things, but it also tends to get super complex super quick. One thing I'd be catious of is hard-coding chat participants. I like the idea of sending people to a general chat where they can escalate as needed.

Another idea is to have an inline Slack chat. Support tools do this, where you can pop a little window and start chatting...

Better Slack integration is on the "Icebox" backlog, but definitely coming.